tests/contrib/django/test_django_appsec.py

```diff
@@ -1,4 +1,9 @@
 import json
+from sys import version_info
+if version_info.major >= 2:
+    from urllib.parse import urlencode
+else:
+    from urllib import urlencode
```

Comment:
This can be imported from `internal.compat` as well

Reply:
ditto